packages/components/src/button/deprecated.tsx

```diff
@@ -29,7 +29,6 @@ function UnforwardedIconButton(
 	deprecated( 'wp.components.IconButton', {
 		since: '5.4',
 		alternative: 'wp.components.Button',
-		version: '6.2',
```

Comment:
I'm removing the version for the IconButton. It seems like a very small wrapper and it's a bit hard to search in wpdirectory about potential usage.